packages/ffe-account-selector-react/src/subcomponents/input-field/InputField.js

```diff
@@ -2,15 +2,15 @@
 /* eslint jsx-a11y/role-has-required-aria-props:0 */
 import React, { Component } from 'react';
 import { func, string, bool, number } from 'prop-types';
+import autoBind from 'react-auto-bind';
 import KryssIkon from '@sb1/ffe-icons-react/lib/kryss-ikon';
+import ChevronIkon from '@sb1/ffe-icons-react/lib/chevron-ikon';
+import classNames from 'classnames';
 
 class Input extends Component {
     constructor(props) {
         super(props);
-        this.onChange = this.onChange.bind(this);
-        this.onFocus = this.onFocus.bind(this);
-        this.onBlur = this.onBlur.bind(this);
-
+        autoBind(this);
```

Comment:
If you just use class field initializers to declare your callbacks with arrow functions, you don't need to bind at all.